src/assets/locales/en/translation.json

```diff
@@ -383,7 +384,8 @@
     "refund_vault_to_address": "back to your address",
     "warning_mbtc_wallets": "Warning: Some Bitcoin wallets display values in mBTC. Please ensure you send the correct amount:",
     "tooltip_bridge_fee": "The bridge fee is paid to the Vaults, Relayers, and maintainers of the system.",
-    "tooltip_security_deposit": "The security deposit is a denial-of-service protection for Vaults that is refunded to you after the minting process is completed."
+    "tooltip_security_deposit": "The security deposit is a denial-of-service protection for Vaults that is refunded to you after the minting process is completed.",
+    "tooltip_transaction_fee": "Transaction Fee"
```

Comment:
@nud3l 
Please update me on proper message text.

Reply:
I think just "Transaction fee" is fine. If you want to add more text: "The fee for the transaction to be included in the parachain."

Reply:
"The fee for the transaction to be included in the parachain." looks better to me as it's a tooltip.